scio-examples/src/main/scala/com/spotify/scio/examples/extra/CloudSqlExample.scala

```diff
@@ -0,0 +1,75 @@
+package com.spotify.scio.examples.extra
+
+import java.sql.{PreparedStatement, ResultSet}
+
+import com.spotify.scio.ContextAndArgs
+import com.spotify.scio.jdbc._
+import org.apache.beam.examples.common.ExampleCloudSqlOptions
+import org.apache.beam.sdk.options.PipelineOptionsFactory
+import org.apache.beam.sdk.values.KV
+
+// Read from cloud sql database table and write to different table in the same database.
+object CloudSqlExample {
+  def main(cmdlineArgs: Array[String]): Unit = {
+    // Ensure this has been called with required database connection details.
+    val cloudSqlOptions: ExampleCloudSqlOptions = PipelineOptionsFactory.fromArgs(cmdlineArgs: _*)
+      .withValidation()
+      .as(classOf[ExampleCloudSqlOptions])
+
+    val (sc, args) = ContextAndArgs(cmdlineArgs)
+    val connOpt = getDbConnectionOptions(cloudSqlOptions)
+    val readOptions = getReadOptions(connOpt)
+    val writeOptions = getWriteOptions(connOpt)
+
+    sc.withName("Read from Cloud Sql").cloudSqlSelect(readOptions)
+      .map(kv => KV.of(kv.getKey + "_", kv.getValue)) // append _ to words
+      .withName("Write to Cloud Sql").saveAsCloudSql(writeOptions)
+    sc.close().waitUntilFinish()
```

Comment:
All other examples don't have `.waitUntilFinish()` so better to leave it out? It's really specific to our internal uses.